src/panoptes/pocs/focuser/birger.py

```diff
@@ -63,10 +63,13 @@ def __init__(self,
                  model='Canon EF-232',
                  initial_position=None,
                  dev_node_pattern='/dev/tty.USA49*.?',
+                 max_command_attempts=5,
                  *args, **kwargs):
         super().__init__(name=name, model=model, *args, **kwargs)
         self.logger.debug('Initialising Birger focuser')
 
+        self.max_command_attempts = max_command_attempts
```

Comment:
@JAAlvarado-Montes I thinks this should be a private variable, `self._max_command_attempts`